### DISCUSSION TITLE
Literature on lightness contrast

### DISCUSSION THREAD

**Discussion by xi** [Myndex/SAPC-APCA#81] [Q&A] [answered]
In the literature I found some evidence that luminance contrast is a good predictor for perceived contrast for people with low vision (emphasis mine):

> Scientific research has shown that for normally sighted subjects, reading rates for
> high colour contrast are as fast as those for high luminance contrast. Data indicates
> that readers rely on information conveyed by colour contrast or luminance contrast,
> whichever yields the best performance. On the other hand, **people with low vision
> read text faster with luminance contrast than colour contrast**. This was true at
> maximum contrast but increasingly so at lower contrasts (Legge et al, 1990).
>
> […]
>
> Luminance contrast is the difference in luminance form two adjacent surfaces. Luminance
> is the amount of reflected light from a colour, or is a measure of clarity; in the same
> illuminance or brightness, dark areas have a lower luminance than bright surfaces.
> Luminance is expressed in candela/m².
>
> Colour contrast is the difference in chromaticity of two adjacent surfaces (with the same luminance).
> 
> -- https://www.anec.eu/images/Publications/technical-studies/ANEC-final-report-1503-1700-Lenoir-et-al.pdf

As far as I understand, APCA is not a luminance contrast, but a lightness contrast. In other words, it is based not on the physical amount of light but on the perceived amount of light.

I am interested to learn more about this. Is there literature that compares luminance contrast to lightness contrast?

**Comment by Myndex** [comment, marked as answer]
> _As far as I understand, APCA is not a luminance contrast, but a lightness contrast. In other words, it is based not on the physical amount of light but on the perceived amount of light._
> _I am interested to learn more about this. Is there literature that compares luminance contrast to lightness contrast?_

### Lightness contrast (Lc) **is** perceptually-uniform luminance contrast.

The importance and implications of perceptual uniformity in creating a model of human perception can not be underestimated. It is why it is spurious to attempt to abstract the math and remove features to equations that are intended to create a uniform scale or metric of a given perception.

I have discussed this extensively in the white papers and articles, all of which you can access here: [git.myndex.com](https://git.myndex.com)

Read the documentation and ask questions, though at the moment, I'm having some difficulty in understanding what you are asking.

LUMINANCE: is a measure of light in the physical world.

LIGHTNESS (and BRIGHTNESS): These are perceptions of light, but there is not a linear relationship to the measure of light, i.e. luminance.

An attempt to remove aspects of a formula that are designed to be perceptually uniform, i.e. ro respond in a manner that is _linear to perception_, is a deeply flawed approach. Doing so ignores the entire purpose of the model.

**Reply by xi** [reply]
I have read a lot of your documentation and I am really trying to understand it. For example this one:

> Luminance is a *linear* measure of light and it is **not** perceptually uniform relative to human visual perception of lightness/darkness/brightness.
> -- https://gist.github.com/Myndex/b21c2621072b93d3a7c1ef8939bc2adb

**Reply by Myndex** [reply]
I do not understand your question?

**Comment by Myndex** [comment]
@xi 

LUMINANCE is a photometric measure of LIGHT. Light in the real world is simple and additiive.

THAT IS: if you have 100 photos and you triple it, you then have 300 photons.

HOWEVER, this is not PERCEPTION. Perception is NOT linear.

If you see 100 photons, and then it is increased to 300 photons, you do not see a tripling, you see only a modest increase.

Perceptual lightness/darkness/brightness are functions of the human vision system. They can not be directly measured. However they can be modeled and therefore reasonably predicted.

Perceptual lightness such as L* (Lstar) makes a non-linear association with the linear light (luminance) and the resultant perception of lightness.

This is all very academic. If you would like to know more, then I highly recommend:

- "Color Appearance Models" by Fairchild

- "Colorimetry" by Schanda

- "Reproduction of Colour" and also "Measuring Colour" by Hunt


In particular, "Reproduction of Colour" by Hunt I find very easy to read, and I really like the way Hunt describe what is a very difficult field. If you are having difficulty understanding my writing (my bad) then try Hunt's books and see if they make more sense.

**Comment by Myndex** [comment]
Additional note to Tobias @xi

Many of your questions do not relate to APCA specifically, but are very general to the overall understanding of the field of visual perception and color appearance modeling.

I think "Reproduction of Colour" by R.W.Hunt is probably where you should start. I have seen some pdfs online at internet libraries.

While I have attempted to include much of the academic background, I also am relying on the bibliographies and it is difficult to determine what someone's level of understanding is here. As you have just started this journey, expect to spend the next year climbing the learning curve.

If you ask questions, I will provide answers, but please refrain from asserting opinions that are not supported by the state of the art. 

**_"Ask Questions" and not "Assert Opinions"_** is the best way to get information from me.
